### DISCUSSION TITLE
nice-article/itmind/navicatzxbwindowspjbjc

### DISCUSSION THREAD

**Discussion by giscus** [itwanger/tobebetterjavaer-giscus#147] [Announcements]
# nice-article/itmind/navicatzxbwindowspjbjc

Navicat 16 Windows 破解版教程

https://tobebetterjavaer.com/nice-article/itmind/navicatzxbwindowspjbjc.html

**Comment by zailushan** [comment]
不错，试了一下 没有问题，谢谢二哥

**Comment by zgw0416** [comment]
二哥，最新的navicat 16.2  好像不能破解了
